public/app/whiteboards/reuse/reuse.html

```diff
@@ -11,18 +11,19 @@ <h2 class="modal-title">Add asset(s)</h2>
           data-search-options-category="searchOptions.category"
           data-search-options-user="searchOptions.user"
           data-search-options-section="searchOptions.section"
-          data-search-options-type="searchOptions.type"></search>
+          data-search-options-type="searchOptions.type"
+          data-ng-if="isSearch || pinned.length || assets.length"></search>
```

Comment:
If the condition above is false then show message, "Sorry, this course has no assets", and hide search options.